schema.graphql

```diff
@@ -5,39 +5,27 @@ schema {
 
 type AccountAddressCreate {
   user: User
-  accountErrors: [AccountError!]!
```

Comment:
Is this formatting on purpose?

Reply:
It is copy-paste formatting from the backend. I'll discuss which one is better 👍

Reply:
As discussed, I've added `schema.graphql` to `.prettierignore` to prevent schema from unintended formatting in future and persist it the same as on backend.